src/EnergyPlus/MixedAir.cc

```diff
@@ -3279,6 +3279,155 @@ namespace MixedAir {
 						SetupEMSActuator( "Outdoor Air Controller", OAController( OAControllerLoop ).Name, "Air Mass Flow Rate", "[kg/s]", OAController( OAControllerLoop ).EMSOverrideOARate, OAController( OAControllerLoop ).EMSOARateValue );
 					}
 
+					VentMechObjectNum = OAController( OAControllerLoop ).VentMechObjectNum;
+					if ( VentMechObjectNum > 0 ){
```

Comment:
Also should add `&& thisAirLoop > 0` here to protect against an airloop not found above.